crates/matrix-sdk/src/sliding_sync/list/request_generator.rs

```diff
@@ -154,7 +152,7 @@ pub(super) fn create_range(
     desired_size: u32,
     maximum_number_of_rooms_to_fetch: Option<u32>,
     maximum_number_of_rooms: Option<u32>,
-) -> Result<(UInt, UInt), Error> {
+) -> Result<RangeInclusive<u32>, Error> {
```

Comment:
```suggestion
) -> Result<RangeInclusive<Bound>, Error> {
```

Thoughts?

Reply:
Agreed with this one, as it's in the function's signature :+1: